contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -216,6 +218,15 @@ public enum OperationalScheme {
 	@Comment("Store planned unshared drt route as a link sequence")
 	public boolean storeUnsharedPath = false; // If true, the planned unshared path is stored and exported in plans
 
+
+	public enum SimulationType {
```

Comment:
As mentioned, @michalmac, could that not be merged with the speed-up functionality, i.e. just "teleportation" could be a third option in this enum which refers to the speed-up approach.